RHDEVDOCS-5940: Draft content for Argo CD CR properties and Repo Server table (#81201)
openshift/openshift-docs: modules/gitops-repo-server-properties.adoc

|`serviceaccount` |`""` |The name of the `serviceaccount` to use with the repo-server pod.

> |`verifytls` |`false` |Defines whether to enforce strict TLS checking on all components when communicating with repo server.

Reviewer: ```suggestion
|`verifytls` |`false` |Defines whether to enforce strict TLS checking on all components when communicating with the repo server.
```
Should "repo server" be "repository server"?

Author: No, it should be repo server.

> |`autotls` |`""` |Provider to use for setting up TLS for the repo-server's gRPC TLS certificate. Currently, only the `openshift` value is acceptable.

Reviewer: ```suggestion
|`autotls` |`""` |Provider to use to set up TLS for the repo-server's gRPC TLS certificate. Currently, only the `openshift` value is acceptable.
```

Author: Fixed.

> |`image` | `argoproj/argocd` |The container image for Argo CD Repo server. This overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.

Reviewer: ```suggestion
|`image` | `argoproj/argocd` |The container image for Argo CD Repo server. This propery overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.
```

Author: Fixed.

|`version` | same as `.spec.Version` |The tag to use with the Argo CD Repo server.
|`logLevel` | `info` |The log level used by the Argo CD Repo server. Valid options are `debug`, `info`, `error`, and `warn`.
|`logFormat` | `text` |The log format to be used by the Argo CD repo server. Valid options are `text` and `json`.

> |`execTimeout` | `180` |Execution timeout in seconds for rendering tools (for example Helm or Kustomize).

Reviewer: ```suggestion
|`execTimeout` | `180` |Execution timeout in seconds for rendering tools, for example, Helm or Kustomize.
```

Author: Fixed.

> |`env` | `__<empty>__` |Environment to set for the repository server workloads.

Reviewer: ```suggestion
|`env` | `__<empty>__` |The environment to set for the repository server workloads.
```
Should "repository server" be "Repo server"?

Author: Yes, it should be "Repo server". Fixed.

> |`enabled` | `__<empty>__` |Flag to enable repo server during Argo CD installation.

Reviewer: ```suggestion
|`enabled` | `__<empty>__` |Flag that enables the Repo server during Argo CD installation.
```

Author: Fixed.

> |`extraRepoCommandArgs` | `__<empty>__` | Pass command line arguments to the repo server workload. They are added to the existing list of arguments set by the Operator.

Reviewer: ```suggestion
|`extraRepoCommandArgs` | `__<empty>__` | Passes command-line arguments to the Repo server workload. The command-line arguments are added to the list of arguments set by the Operator.
```

Author: Fixed.

> |`initContainers` | `__<empty>__` |The number of `init` containers in the ArgoCD Application Controller component. This field is optional.

Reviewer: ```suggestion
|`initContainers` | `__<empty>__` |The number of `init` containers in the Argo CD Application Controller component. This field is optional.
```

Author: Fixed.

> |`sidecarContainers` | `__<empty>__` |The number of `sidecar` containers in the ArgoCD Application Controller component. This field is optional.

Reviewer: ```suggestion
|`sidecarContainers` | `__<empty>__` |The number of `sidecar` containers in the Argo CD Application Controller component. This field is optional.
```

Author: Fixed.

> |`volumes` | `__<empty>__` |Configures additional volumes used for the repo server deployment. This field is optional.

Reviewer: ```suggestion
|`volumes` | `__<empty>__` |Configures additional volumes used for the Repo server deployment. This field is optional.
```

Author: Fixed.

> |`volumeMounts` | `__<empty>__` |Configures additional volume mounts used for the repo server deployment. This field is optional.

Reviewer: ```suggestion
|`volumeMounts` | `__<empty>__` |Configures additional volume mounts used for the Repo server deployment. This field is optional.
```

Author: Fixed.

|`replicas` | `__<empty>__` |The number of replicas for the Argo CD repo server. Must be greater than or equal to `0`.

> |`remote` | `__<empty>__` |Specifies the remote URL of the repo server container.

Reviewer: ```suggestion
|`remote` | `__<empty>__` |Specifies the remote URL of the Repo server container.
```

Author: Fixed.

> |`logFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are `text` or `json`.

Reviewer 2: Again, I am not sure about the capitalization here: "Argo CD Repo server"

Author: Fixed.

> |`logFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are `text` or `json`.

Reviewer 2: ```suggestion
|`logFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are `text` and `json`.
```
Small nit.

Author: Fixed.

|`execTimeout` | `180` |Execution timeout in seconds for rendering tools (for example Helm or Kustomize).
|`env` | `__<empty>__` |Environment to set for the repository server workloads.

> |`enabled` | `__<empty>__` |Flag to enable Repo Server during Argo CD installation.

Reviewer 2: Same comment about repo server capitalization

Author: Fixed.

> |`extraRepoCommandArgs` | `__<empty>__` | Pass command line arguments to the repo server workload. They are added to the existing list of arguments set by the Operator.

Reviewer 2: Same comment about repo server capitalization

> |`replicas` | `__<empty>__` |The number of replicas for the Argo CD Repo server. Must be greater than or equal to `0`.

Reviewer 2: Same comment about repo server capitalization

Author: Fixed.

> |`remote` | `__<empty>__` |Specifies the remote URL of the Repo Server container.

Reviewer 2: Same comment about repo server capitalization
